sourcegraph/about
# Update iframe videos

Removes `data-cookieconsent="ignore"` from iframed videos.

## To Test
- Check records in PR in the deploy environment in incognito mode and ensure that they load properly.
